notebooks/supernovae_plus_extended_source_tutorial.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<p>can you make the observed times to be printed on the images? Also, might be good having a shared color scale</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>